python/python/glide/async_commands/redis_modules/search/commands.py

```diff
@@ -0,0 +1,96 @@
+from typing import List, Optional, cast
+
+from glide.async_commands.redis_modules.search.optional_params import (
+    FieldFlags,
+    Frequencies,
+    Highlights,
+    InitialScan,
+    Offset,
+)
+from glide.constants import TOK
+from glide.protobuf.redis_request_pb2 import RequestType
+from glide.redis_client import TRedisClient
+
+from .field import Field
+from .index import Index
+
+
+class Search:
+    """Class for RediSearch commands."""
+
+    async def create_index(
+        self,
+        client: TRedisClient,
+        index: Index,
+        schema: List[Field],
+        max_text_fields: bool = False,
+        temporary_seconds: Optional[int] = None,
+        offset: Offset = Offset.STORE_OFFSET,
+        highlight: Highlights = Highlights.USE_HIGHLIGHTS,
+        field_flags: FieldFlags = FieldFlags.USE_FIELDS,
+        frequencies: Frequencies = Frequencies.SAVE_FREQUENCIES,
+        stop_words: Optional[List[str]] = None,
+        initial_scan: InitialScan = InitialScan.SCAN_INDEX,
+    ) -> TOK:
+        """
+        Creates a new search index with the specified configuration.
+        See https://redis.io/commands/ft.create/ for more details.
+
+        Args:
+            client (TRedisClient): The Redis client to execute the command.
+            index (Index): The definition of the index, including its name and type.
+            schema (List[Field]): The schema of the index, specifying the fields and their types.
+            max_text_fields (bool): Forces RediSearch to encode indexes as if there were more than 32 text attributes.
+                Additional attributes (beyond 32) can be added using FT.ALTER.
+            temporary_seconds (Optional[int]): Creates a lightweight temporary index that automatically expires after a set period of inactivity, measured in seconds.
+                The index's internal idle timer resets each time a search or addition operation occurs.
+                Because such indexes are lightweight, you can create thousands of such indexes without negative performance implications and, therefore,
+                you should consider using SKIPINITIALSCAN to avoid costly scanning.
+                Warning: When temporary indexes expire, they drop all the records associated with them.
+            offset (Offset): Whether to store term offsets for documents.
+            highlight (Highlights): Whether to disable / enable highlighting support.
+            field_flags (FieldFlags): Whether to store attribute bits for each term.
```

Comment:
re. the commend on FieldFlags below = make sure to change field_flags to fields_flag too